docs/adr/0061-integration-de-typescript-dans-l-api.md

```diff
@@ -0,0 +1,72 @@
+# 61. Intégration de TypeScript dans l'API
+
+Date : 2025-05-06
+
+## État
+
+En cours
+
+## Contexte
+
+Côté API, nous avons une base de code JavaScript de près 120 000 lignes et de plus de 9 ans.
+
+Plusieurs réflexions ont été menées sur la mise en place de TypeScript, notamment lors des TechDays 2022, suite à un sondage global côté Engineering où la conclusion était d’aller vers l’intégration progressive de TypeScript.
```

Comment:
Dans le contexte de l'ADR, j'aurais trouvé ça intéressant de remettre les points les plus importants qui pousse à ce choix et à cette conclusion

Reply:
Ainsi que les contre qui ont pu être noté, histoire de ne pas refaire les discussions éparpillés

Reply:
Je reprendrais les points de décisions suite au TechTime et au sondage.

Reply:
J'ai ajouté les besoins originaux des tech days + ceux évoqués par @Steph0.